docs/sources/developers/http_api/reporting.md

```diff
@@ -15,10 +15,315 @@ title: Reporting API
+| timeZone           | string    | Time zone used to schedule report execution.                                                                                                                                                                                                                                                                                                                                                                                                      |
+| orientation        | string    | Can be `portrait` or `landscape`.                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| layout             | string    | Can be `grid` or `simple`.                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| enableDashboardUrl | bool      | Adds a dashbaord url to the bottom of the report email.                                                                                                                                                                                                                                                                                                                                                                                           |
+| formats            | []string  | Specified what kind of attachment to generate for the report - `csv`, `pdf`, `image`. `pdf` is the default one.                                                                                                                                                                                                                                                                                                                                   |
+| dashboards         | []object  | Dashboards to generate a report for.<br/> [Dashboard object schema](#dashboard-schema).                                                                                                                                                                                                                                                                                                                                                           |
+
+#### Dashboard Schema
```

Comment:
```suggestion
#### Report dashboard Schema
```
We call it Report dashboard to differentiate from the actual dashboard data, as this one is specifically for reports. Not sure i this is relevant for the docs though.